proxyd/metrics.go

```diff
@@ -504,6 +512,10 @@ func RecordCacheMiss(method string) {
 	cacheMissesTotal.WithLabelValues(method).Inc()
 }
 
+func RecordRpcRequestCount(method string, count int) {
+	rpcReqTotal.WithLabelValues(method).Add(float64(count))
+}
+
```

Comment:
Is it possible to use the Inc() function instead of Add(float64()) as it is more inline with the current proxyd style
```
rpcReqTotal.WithLabelValues(method).Inc()
```